### DISCUSSION TITLE
Thorium OS for ChromeOS - CHROMEBOOKS

### DISCUSSION THREAD

**Discussion by FacelessKiller** [Alex313031/ThoriumOS#50] [Ideas]
Can you try to create a version of ThoriumOS using [this link](https://dl.google.com/dl/edgedl/chromeos/recovery/chromeos_14989.107.0_snappy_recovery_stable-channel_mp.bin.zip)

https://chrome100.dev/board/snappy/

**Comment by Alex313031** [comment]
@FacelessKiller Ahh, every officially released board with a Codename has an "overlay" with board specific drivers and config files. For example, the one for Chrome OS Flex is called "reven" and includes stuff to maximize compatibility with regular hardware, which is what ThoriumOS is based on.

That said, I don't think many people realize the amount of development work, and the amount of compiling time even with a powerful computer like mine, it takes to build ChromiumOS and make a bootable image out of it.

Changing the overlay to be based on the Snappy  overlay is kind of a no-go for me, doing all that work just to support the small subset of users (and you may be the only one), who are trying to run ThoriumOS on that specific Chromebook.

In any case, have you tried installing ThoriumOS as is?
You have to have Mr.Chromebox firmware anyway to install ChromiumOS/ThoriumOS on a chromebook, and in that case, if ThoriumOS doesn't work, you might try Chrome OS Flex.

**Reply by Interested-Deving-1896** [reply]
@FacelessKiller @Alex313031 

There are other ways that we are working on bringing [compiling / building] Penguins-eggs for ChromiumOS-Based Derivatives.

We can get penguins-eggs to help compile/build each or any overlay into/onto ThoriumOS. Then, with penguins-eggs, we can make both a BIN + IMG + ISO of ThoriumOS.

You can see an overview of other ways located at this Git-Issue . . .

pieroproietti/penguins-eggs#475

**Reply by Interested-Deving-1896** [reply]
https://github.com/chromebrew/chromebrew/discussions/12273

**Reply by SirHumza** [reply]
I have tried installing thoriumOS using the mrchromebox.tech bios on my C214ma chromebook. It doesn't boot. Only shows the grub menu, and keeps loading (a _ in a black screen)
